format.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Would this script and the format changes necessary if we will eventually move the sot to internal codebase where the format will be different?

Reply:
This helps the external contributors to do the formatting before copying the code to internal for review. The format rules should be the same as internal codebase.

Reply:
oh nice! good to know. thanks!